tests/common/dualtor/dual_tor_io.py

```diff
@@ -256,22 +258,23 @@ def generate_from_server_to_t1(self):
             ptf_intf_to_mac_map[ptf_intf] = self.ptfadapter.dataplane.get_mac(0, ptf_intf)
 
         logger.info("-"*20 + "Server to T1 packet" + "-"*20)
-        logger.info("Ethernet address: dst: {} src: {}"
-                    .format(self.vlan_mac,
-                            'random' if self.tor_vlan_intf is None
-                                     else ptf_intf_to_mac_map[
-                                         self.tor_to_ptf_intf_map[self.tor_vlan_intf]
-                                         ]
-                           )
-                    )
-        logger.info("IP address: dst: {} src: {}"
-                    .format('random',
-                            'random' if self.tor_vlan_intf is None
-                                     else self.ptf_intf_to_server_ip_map[
-                                                   self.tor_vlan_intf
-                                               ]
-                           )
-                   )
+        if self.tor_vlan_intf is None:
+            src_mac = 'random'
+            src_ip = 'random'
+        else:
+            ptf_port = self.tor_to_ptf_intf_map[self.tor_vlan_intf]
+            src_mac = ptf_intf_to_mac_map[ptf_port]
+            src_ip = self.tor_to_ptf_intf_map[ptf_port]
```

Comment:
I believe there is an additional dictionary lookup to get the server IP, in `self.ptf_intf_to_server_ip_map`